ui2/src/pages/network/Distribution.tsx

```diff
@@ -40,21 +41,24 @@ export const DistributionServiceTable = () => {
   const services = [...auctionServices, ...biddingServices];
 
   return (
-    <StripedTable
-      headings={['Service', 'Operator', 'Provider', 'Consumer', 'Role']}
-      loading={biddingServicesLoading || auctionServicesLoading}
-      rows={services.map(c => {
-        return {
-          elements: [
-            getTemplateId(c.templateId).split('.')[2],
-            getName(c.payload.operator),
-            getName(c.payload.provider),
-            getName(c.payload.customer),
-            party === c.payload.provider ? 'Provider' : 'Consumer',
-          ],
-        };
-      })}
-    />
+    <>
+      <ActionTile actions={[{ path: '/app/distribution/new', label: 'New Distribution' }]} />
```

Comment:
Should this path be `/app/setup/distribution/new/auction`?

I wonder if its time to have a global list of app paths somewhere, too...

Reply:
Ah yeah, I see now that path in `App.tsx` uses the same component as `Auction` just named differently...will change. Agreed that a global app path list would be good!